lightning/src/ln/channelmanager.rs

```diff
@@ -2421,6 +2421,9 @@ where
 	/// See `PendingOutboundPayment` documentation for more info.
 	///
 	/// See `ChannelManager` struct-level documentation for lock order requirements.
+	#[cfg(test)]
+	pub(super) pending_outbound_payments: OutboundPayments,
+	#[cfg(not(test))]
```

Comment:
Could we test with `list_recent_payments` instead of exposing this?

Reply:
Yeah true, I have to expose this in #3440 anyway but may as well use the public API where possible.